Makefile

```diff
@@ -379,7 +385,7 @@ node_modules/angular/angular.min.js: .build/node_modules.timestamp
 		-e 's|/@?main=$*.js|$*.js|' \
 		-e '/default\.js/d' \
 		-e 's|\.\./utils/watchwatchers.js|lib/watchwatchers.js|' \
-		-e '/$*.js/i\    <script src="lib/ngeo.js"></script>' $< > $@
+		-e '/$*.js/i\'$$'\n''    <script src="lib/ngeo.js"></script>'$$'\n' $< > $@
```

Comment:
You're right, it's possible to remove the quotes.
However, since `$'\n` is the new line character, I need to double the `$` to make it work in the Makefile.